    /** Do 1.
     * @throws TestException1 when error occurrs.
     */
    public void doStuff1() throws TestException1 {
        try {
            doStuff2();
        } catch (final TestException2 e) { }
        throw new Input_02().new TestException1("");
    }

